README.md

```diff
@@ -1237,3 +1237,32 @@ DELETE /api/v1/users/mary
 ```
 HTTP/1.1 200 OK
 ```
+
+# Upgrades
+
+Initiates an upgrade to the provided version. Once this endpoint is called [Horticulturalist](https://github.com/medic/horticulturalist) will take over and complete the installation.
+
+### Permissions
+
+`_admin` (CouchDB Administrator)
+
+## Example
+
+```
+POST /api/v1/upgrade
+{
+  "build": {
+    "namespace": "medic",
+    "application": "medic",
+    "version": "3.0.0-beta.1"
+  }
+}
+```
+
```

Comment:
> Seems more logical to me to keep it more simple and flexible as a first cut and lock it down further later when these needs become more clear

Heh, that is the direct opposite of what I think! As in, I think we should hard-code stuff so we're not building in flexibility we have to make decisions about until we need that flexibility. Specifically , like the last three bullet points in the comment above. We'd have to make decisions about that, consider security etc. I'd rather Just Not for now, at least until we've used it for a couple of months.